testfiles/task.lkt

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="UTF-8"?>

```

Comment:
I'll have a look at the test file and read up about the principle and then create the class. All new to me.